.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -10,13 +10,44 @@
   - when this PR is merged.
   -->
 
+## How to test this PR?
```

Comment:
Similarly, let's follow "Proof that changes are correct", but in the explanatory comment, you can add something about also explaining how the reviewer can verify the work. You can also say: for UI-related PRs, add screenshots and recordings (i.e. similar to what you wrote below) for both LTR and RTL layouts.

Also, I have a question. If the developer posts screenshots/recordings/etc., are the verification instructions still needed?

Reply:
Actually verifications instructions is important considering there are cases in app where we have to follow specific instructions to reach a certain screen. For example, opening a concept card, viewing hints and solution, etc.

Images and for proof that its correct and for very easy PR they help in merging the PR just by looking at the code and screenshots but for feature based PRs screenshots help in getting clear idea what to expect and instructions help in trying it out on own and cross verifying it in various situations like rotating the app, talkback, etc.

Reply:
Ah ok. Is this only for UI-based PRs? If so, you can add a section at the bottom for that (it should probably come before the Espresso tests).

If not, then how is this section supposed to be filled in for non-UI-based PRs?

Reply:
Mostly it UI based PRs only. Like opening a concept card, etc.

Added at bottom 
```

### How to test this PR?
<!-- Explain what exact steps reviewer should perform to test this PR. -->
```